<launch>
  <arg name="image_raw_topic" default="/real_camera"/>
  <arg name="signal_cmd_topic" default="/signal_vel"/>
  <arg name="mask_mode" default="False"/>

    <node name="signal_panel_recognition" pkg="signal_recognition" type="signal_panel_recognition.py" output="screen">
      <param name="image_raw_topic" value="$(arg image_raw_topic)"/>
      <param name="signal_cmd_topic" value="$(arg signal_cmd_topic)"/>
      <param name="mask_mode" value="$(arg mask_mode)"/>
    </node>

</launch>